types/k6/net/grpc.d.ts

```diff
@@ -29,7 +29,7 @@ export interface Params {
 
     tags?: object;
 
-    timeout?: string;
+    timeout?: number;
```

Comment:
This can be a number (milliseconds) _or_ a string.

Reply:
Ok thanks. Fixed https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57368/commits/130b6f3073d5d6fbb1e69c26c86f3e32fac00eed

Reply:
@na--, also `ConnectParams.timeout` ?